### ISSUE TITLE
Close icon in PDF viewer

### ISSUE THREAD

**Issue by mammutserver** [nextcloud/server#22122]
The close icon on the PDF viewer has a weird behavior in my browser(s). I tested it with Chrome,Firefox, Opera and Safari on several devices.

Server version: 19.0.1
PHP version: 7.4.7
Webserver: NGINX

Further information can be provided on request.

![image](https://user-images.githubusercontent.com/46828369/89506645-b8fad280-d7cb-11ea-9a83-f972d0f54531.png)

**Comment by j-ed** [comment]
I cannot replicate your problem with Firefox. It looks fine to me:

![grafik](https://user-images.githubusercontent.com/2572764/89510296-aafb8080-d7d0-11ea-9d6c-6c6b193ee6a2.png)

BTW, this is the official bug tracker of the project development and not the [Nextcloud help forum](https://help.nextcloud.com/, which has explicitly been created to address such problems first.

**Comment by szaimen** [comment]
I have the same issue:
Server is running on:
Ubuntu 20.04 LTS
PHP 7.4
Apache2

Clientbrowser:
Chromium 84 based Browser on Windows 10

**Comment by juliusbr** [comment]
For me it only looks weird if system scaling is set to 150%, either 100% or 200% work fine.

**Comment by szaimen** [comment]
Yeah, it seems to be a scaling issue. On my Microsoft Surface Pro 3 it looks strange by default (probably 150% scaling by default.)
Here is a video of it.
[Screen Recording.zip](https://github.com/nextcloud/server/files/5078720/Screen.Recording.zip)

**Comment by j-ed** [comment]
@juliusbr I can confirm this behavior with Firefox 79.0 and the zoom levels 120-170, up to the level 110 and with the level 200, everything is correctly displayed again.
